services/article.js

```diff
@@ -1,107 +1,129 @@
 const {
   Article,
   ArticleLinksTag,
+  Comment,
   Like,
   Tag,
-  View,
+  User,
   sequelize
 } = require('../models');
 
 exports.getArticleList = async (category, pageNumber) => {
   const CONTENTS_PER_PAGE = 10;

```

Comment:
Maybe we can put this phrase out of this scope, cuz it's an unchangeable value.

Reply:
LGTM

Reply:
Should constant value alway be outside of the scope? CONTENT_PER_PAGE is used only in this function so that I think it is unnecessary to put this phrase out of the scope.